DigitalLearningSolutions.Data/Models/User/AdminUser.cs

```diff
@@ -2,5 +2,26 @@
 {
     public class AdminUser : User
     {
+        public bool CentreAdmin { get; set; }
```

Comment:
Hmm.. I'm wondering whether it's worth making the name consistent at least at a class level - e.g. isCentreManager, isCentreAdmin etc

Reply:
I was trying to go for consistency with the database column names, so they match as closely as possible to the relevant table (excluding shared columns like Id etc.). Do we still want to change the bools to "is..." when they don't match the table?